### PR TITLE
favicon

### PR DESCRIPTION
**Pull request by Javitocatral**

### CODE REVIEW

File: static/v-pink-vermut.png
Comment (zwergius) [inline static/v-pink-vermut.png:1]: This should be smaller. The old favicon was 1.53kb this is 63.7kb

**Review by zwergius — CHANGES_REQUESTED** [review]

**Comment by zwergius** [comment]
1) This is no longer working, if you review your own PR you will see why

2) Now that you used the generator, I was expecting more favicon than 1, where is the rest ?

**Comment by zwergius** [comment]
The favicons are poor quality, surely you have not used the original svg to generate them.

Also they are not in use.

**Comment by Javitocatral** [comment]
I cut out the original png and took the V from there, I passed it through the application that you gave me. I don't know what else you want me to do with it. I don't know how to make svg to get a V because it doesn't exist in the material that you gave me.

**Comment by zwergius** [comment]
You have added many files here but still only using 1 of the icons. I recommend you study how favicon work

File: src/app.html [hunk 1]
```diff
@@ -4,7 +4,13 @@
     <!-- <meta charset="utf-8" /> -->
     <meta http-equiv="content-type" content="text/html; charset=utf-8" />
 
-    <link rel="icon" href="%sveltekit.assets%/favicon.png" />
+    <link rel="icon" href="%sveltekit.assets%/favicon.ico" />
+    <link rel="icon" type="image/png" sizes="32x32" href="%sveltekit.assets%/favicon-32x32.png" />
+    <link rel="icon" type="image/png" sizes="16x16" href="%sveltekit.assets%/favicon-16x16.png" />
+    <link rel="sitemap" type="application/xml" href="%sveltekit.assets%/sitemap.xml" />
+    <link rel="apple-touch-icon" sizes="180x180" href="%sveltekit.assets%/apple-touch-icon.png" />
+    <link rel="mask-icon" href="%sveltekit.assets%/safari-pinned-tab.svg" color="#da532c" />
```
Comment (zwergius) [inline src/app.html:12]: Think you deleted this file in the last commit

File: src/app.html [hunk 2]
```diff
@@ -4,7 +4,13 @@
     <!-- <meta charset="utf-8" /> -->
     <meta http-equiv="content-type" content="text/html; charset=utf-8" />
 
-    <link rel="icon" href="%sveltekit.assets%/favicon.png" />
+    <link rel="icon" href="%sveltekit.assets%/favicon.ico" />
+    <link rel="icon" type="image/png" sizes="32x32" href="%sveltekit.assets%/favicon-32x32.png" />
+    <link rel="icon" type="image/png" sizes="16x16" href="%sveltekit.assets%/favicon-16x16.png" />
+    <link rel="sitemap" type="application/xml" href="%sveltekit.assets%/sitemap.xml" />
```
Comment (zwergius) [inline src/app.html:10]: Did you add a sitemap?

File: src/app.html [hunk 3]
```diff
@@ -5,11 +5,11 @@
     <meta http-equiv="content-type" content="text/html; charset=utf-8" />
 
     <link rel="icon" href="%sveltekit.assets%/favicon.ico" />
-    <link rel="icon" type="image/png" sizes="32x32" href="%sveltekit.assets%/favicon-32x32.png" />
-    <link rel="icon" type="image/png" sizes="16x16" href="%sveltekit.assets%/favicon-16x16.png" />
-    <link rel="sitemap" type="application/xml" href="%sveltekit.assets%/sitemap.xml" />
-    <link rel="apple-touch-icon" sizes="180x180" href="%sveltekit.assets%/apple-touch-icon.png" />
-    <link rel="mask-icon" href="%sveltekit.assets%/safari-pinned-tab.svg" color="#da532c" />
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
```
Comment (zwergius) [inline src/app.html]: Now you removed this again %sveltekit.assets%

File: src/app.html [hunk 4]
```diff
@@ -5,10 +5,10 @@
...
 
     <link rel="icon" href="%sveltekit.assets%/favicon.ico" />
-    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
-    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
-    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
-    <link rel="manifest" href="/site.webmanifest" />
+    <link rel="apple-touch-icon" sizes="180x180" href="%sveltekit.assets%/apple-touch-icon.png" />
+    <link rel="icon" type="image/png" sizes="32x32" href="%sveltekit.assets%/favicon-32x32.png" />
+    <link rel="icon" type="image/png" sizes="16x16" href="%sveltekit.assets%/favicon-16x16.png" />
+    <link rel="manifest" href="%sveltekit.assets%/site.webmanifest" />
```
Comment (Javitocatral) [inline src/app.html:11]: in the first pr header fixes if they are removed I realized and I corrected it quickly

File: src/app.html (hunk 3 above)
Comment (Javitocatral) [reply src/app.html]: in the first pr header fixes if they are removed I realized and I corrected it quickly

**Pull request merged**